### PR TITLE
Added support for continuous mode (android only)

### PR DESCRIPTION
**Pull request by fttx**
https://github.com/phonegap/phonegap-plugin-barcodescanner/issues/69

### CODE REVIEW

**Comment by baversjo** [comment]
@fttx This is very useful for me as well.
However I cannot get it to work. The official version works though. Maybe you need to update something, or I'm doing something wrong?

I tried the following:
```
cordova plugin remove phonegap-plugin-barcodescanner
cordova plugin add @fttx/phonegap-plugin-barcodescanner
```

but get this error during `cordova run android`:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.
> Could not find :barcodescanner:.
  Searched in the following locations:
      file:/Users/johan/Library/Android/sdk/extras/m2repository//barcodescanner//barcodescanner-.pom
      file:/Users/johan/Library/Android/sdk/extras/m2repository//barcodescanner//barcodescanner-.aar
      file:/Users/johan/Library/Android/sdk/extras/google/m2repository//barcodescanner//barcodescanner-.pom
      file:/Users/johan/Library/Android/sdk/extras/google/m2repository//barcodescanner//barcodescanner-.aar
      file:/Users/johan/Library/Android/sdk/extras/android/m2repository//barcodescanner//barcodescanner-.pom
      file:/Users/johan/Library/Android/sdk/extras/android/m2repository//barcodescanner//barcodescanner-.aar
      https://jcenter.bintray.com//barcodescanner//barcodescanner-.pom
      https://jcenter.bintray.com//barcodescanner//barcodescanner-.aar
      https://maven.google.com//barcodescanner//barcodescanner-.pom
      https://maven.google.com//barcodescanner//barcodescanner-.aar
      https://repo1.maven.org/maven2//barcodescanner//barcodescanner-.pom
      https://repo1.maven.org/maven2//barcodescanner//barcodescanner-.aar
      file:/Users/johan/projects/smrt-cordova/platforms/android/app/libs/barcodescanner-.aar
      file:/Users/johan/projects/smrt-cordova/platforms/android/app/libs/barcodescanner.aar
  Required by:
      project :app
```

Any ideas?
Thank you

**Comment by fttx** [comment]
I don't know, i've tried to create a new cordova project and add it, and it just worked:

screen shot 2018-01-19 at 21 28 04

Try to remove node_modules, plugins and package-lock.json and do `npm i` again

File: README.md [hunk 1]
```diff
@@ -117,7 +117,8 @@ A full example could be:
           formats : "QR_CODE,PDF_417", // default: all but PDF_417 and RSS_EXPANDED
           orientation : "landscape", // Android only (portrait|landscape), default unset so it rotates with the device
           disableAnimations : true, // iOS
-          disableSuccessBeep: false // iOS and Android
+          disableSuccessBeep: false, // iOS and Android
+          continuosMode: false // Android
```
Comment (macdonst) [inline README.md]: Can you switch this to `continuousMode` to keep the options consistent with English spelling?

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "phonegap-plugin-barcodescanner",
-  "version": "7.0.0",
+  "name": "@fttx/phonegap-plugin-barcodescanner",
```
Comment (macdonst) [inline package.json]: Revert this change.

File: package.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+  "version": "7.0.1",
```
Comment (macdonst) [inline package.json]: Revert this change. The version number of the plugin will be updated when it is published to npm.

File: package.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
   "description": "You can use the BarcodeScanner plugin to scan different types of barcodes (using the device's camera) and get the metadata encoded in them for processing within your application.",
   "cordova": {
-    "id": "phonegap-plugin-barcodescanner",
+    "id": "@fttx/phonegap-plugin-barcodescanner",
```
Comment (macdonst) [inline package.json]: Revert this change.

File: src/android/com/phonegap/plugins/barcodescanner/BarcodeScanner.java [hunk 5]
```diff
@@ -46,6 +50,7 @@
     private static final String PREFER_FRONTCAMERA = "preferFrontCamera";
     private static final String ORIENTATION = "orientation";
     private static final String SHOW_FLIP_CAMERA_BUTTON = "showFlipCameraButton";
+    private static final String CONTINUOS_MODE = "continuosMode";
```
Comment (macdonst) [inline src/android/com/phonegap/plugins/barcodescanner/BarcodeScanner.java]: Switch to:

```
private static final String CONTINUOUS_MODE = "continuousMode";
```

File: src/android/com/phonegap/plugins/barcodescanner/BarcodeScanner.java [hunk 6]
```diff
@@ -65,6 +70,9 @@
     private JSONArray requestArgs;
     private CallbackContext callbackContext;
 
+    private LocalBroadcastManager broadcastManager;
+    private BroadcastReceiver continuosModeBroadcastReceiver;
```
Comment (macdonst) [inline src/android/com/phonegap/plugins/barcodescanner/BarcodeScanner.java]: Switch to `continuousModeBroadcastReceiver`

**Comment by StoneRanger** [comment]
Hi, any plans on having this in the original cordova plugin?
I would like to have this as well.
Greetings

**Comment by fttx** [comment]
@macdonst can you merge it?

**Comment by yuyat** [comment]
Something more is needed apart: continuousMode: true?
Because it does't work.

**Comment by fttx** [comment]
@yuyat there is a dependency issue with the barcodescanner library.
@macdonst i've done the changes you requested, can you merge it?
